CONTRIBUTING_DOCS.md

```diff
@@ -1,18 +1,29 @@
 # Contribute to Documentation
```

Comment:
We moved contributing docs on a [Wiki](https://github.com/openvinotoolkit/openvino/wiki). Should we also move this file to wiki?

CC @ilya-lavrenov

Reply:
[master ](https://github.com/openvinotoolkit/openvino)branch does have this file. 

BTW, I have completely removed `CONTRIBUTING_DOCS.md` because contribution to docs is the same as contribution to all other parts. Docs guidelines are [here](https://github.com/openvinotoolkit/openvino/wiki/CodingStyleGuideLinesDocumentation)